recipes/UniFunc/meta.yaml

```
@@ -0,0 +1,47 @@
+
+{% set name = "unifunc" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PedroMTQ/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 9eaa196505dabe471b3ac790f3e84770919f53728de2f6f0ecbbb550812fde97
+  path: ..
+  
+build:
+  number: 0
+
+requirements:
+  build:
+    - pip
+    - python
+    - setuptools
```

```suggestion
  build:
    - setuptools >=42
    - wheel
  host:
    - pip
    - python >=3.6
```
I assumed that `python >=3.6` is your minimum version, you might need to change that.